docker-compose.yml

```diff
@@ -0,0 +1,45 @@
+version: '3'
+
+services:
+  db:
+    image: mysql:latest
+    volumes:
+      - ./db:/var/lib/mysql
```

Comment:
Maybe I'm mistaken but `./db` refers to the repository root directory `db` -- is that intentional? I haven't actually tested this to see if a `db` directory gets generated or something.

If this isn't what you meant, you'll need to create a volume separately and then refer that here. I.e.

```yml
services:
  db:
    image: mysql:5.7
    volumes:
      - db:/var/lib/mysql

...

volumes:
  db:
```